parm/config/config.resources

```diff
@@ -60,11 +60,12 @@ if [ ${step} = "prep" ]; then
       export memory_prep="40G"
     fi
 
-elif [ ${step} = "aerosol_init" ]; then
+elif [[ "${step}" = "aerosol_init" ]]; then
```

Comment:
I'm guessing these changes are 'linter' related, but just as an FYI, this job is not used when doing Aero DA, this is for non-DA cycling like what is done in the HR prototypes.

Reply:
Yes, this change was driven by the linter check.   We can make whatever change(s) the Aerosol and g-w team want.   I have no preference.

Reply:
I think these changes are fine, just wanted to post an FYI for yours/others awareness